lib/Minz/Configuration.php

```diff
@@ -210,14 +210,22 @@ public function __set(string $key, mixed $value): void {
 	 * Save the current configuration in the configuration file.
 	 */
 	public function save(): bool {
+		$tmp_filename = $this->config_filename . '.tmp';
 		$back_filename = $this->config_filename . '.bak.php';
-		@rename($this->config_filename, $back_filename);
 
-		if (file_put_contents($this->config_filename,
+		if (file_put_contents($tmp_filename,
 			"<?php\nreturn " . var_export($this->data, true) . ';', LOCK_EX) === false) {
```

Comment:
```suggestion
		if (!file_put_contents($tmp_filename,
"<?php\nreturn " . var_export($this->data, true) . ';', LOCK_EX)) {
```

I was confused by the previous syntax, and a return of (int)0 should also be accounted as an error

Reply:
Does that ever happen though?

Reply:
> This function returns the number of bytes that were written to the file, or [false](https://www.php.net/manual/en/reserved.constants.php#constant.false) on failure.

never mind